### PR TITLE
05.Ansible.Run Tasks and playbooks

### PR DESCRIPTION
**Pull request by javid87**
Add 05.Ansible.Run Task. Please check.
Thanks

### CODE REVIEW

File: Javid_Alizada/05.Ansible_run/group_vars/sa_all.yml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#sa_user_name: javid_alizada
+sa_comment: Javid Alizada
+sa_user_pass: $1$I2iQh1ir$I8E8PvBei1.Lzm6Vhf3Rf/
```
Comment (pluhin) [inline Javid_Alizada/05.Ansible_run/group_vars/sa_all.yml]: could you please use vault for that

File: Javid_Alizada/05.Ansible_run/host_vars/deb01.yml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+ansible_host: 10.0.0.201
+ansible_ssh_private_key_file: /home/javid_alizada/DevOps_Traning/vagrant_project/ansible_box/.vagrant/machines/deb01/virtualbox/private_key
```
Comment (pluhin) [inline Javid_Alizada/05.Ansible_run/host_vars/deb01.yml]: could you please use vault for that

File: Javid_Alizada/05.Ansible_run/group_vars/sa_all.yml (hunk 1 above)
Comment (javid87) [reply Javid_Alizada/05.Ansible_run/group_vars/sa_all.yml]: Done Please check . 
Added Vault pass. with  "ansible-vault encrypt group_vars/sa_all.yml" . if you want i can send you my vault pass.
Thanks

File: Javid_Alizada/05.Ansible_run/host_vars/deb01.yml (hunk 2 above)
Comment (javid87) [reply Javid_Alizada/05.Ansible_run/host_vars/deb01.yml]: Done. encrypt files in host_vars with vault.

**Pull request merged**